grails-app/controllers/mama/ng/scheduler/ScheduleController.groovy

```diff
@@ -0,0 +1,117 @@
+package mama.ng.scheduler
+
+import grails.converters.JSON
+
+class ScheduleController {
+
+    static allowedMethods = [save: "POST", update: "PUT", delete: "DELETE"]
+
+    def cronParserService
+
+    def list() {
+        params.max = Math.min(params.max ? params.int('max') : 50, 200)
+        def list = Schedule.list(params)
+        withFormat {
+            json {
+                response.status = 200
+                render list as JSON
+            }
+        }
+    }
+
+    def show(Schedule instance) {
+        withFormat {
+            json {
+                response.status = 200
+                render instance as JSON
+            }
+        }
+    }
+
+    def messages(Schedule instance) {
+        def list = Message.findAllBySchedule(instance)
```

Comment:
I think you're right. I can imagine some crazy situations (a bad schedule fires every minute for a day and fails each time), but maybe let's leave it as is for now.